packages/react/src/index.js

```diff
@@ -10,49 +10,50 @@ const createCss = (init) => {
 	const sheet = createCoreCss(init)
 
 	return assign(sheet, {
-		/** Returns a React component. */
+		/** Returns a React forwarded ref object. */
 		styled: (
 			/** Type of component. */
 			...inits
 		) => {
-			const defaultType = inits.map((init) => (Object(init).type ? init.type : init)).find((init) => init) || 'span'
+			const defaultType = inits.map((init) => (Object(init).stitchesType ? init.stitchesType : init)).find((init) => init) || 'span'
 			const composition = sheet.css(...inits.filter((init) => $$composers in Object(init) || (init && typeof init === 'object' && !init.$$typeof)))
 
-			/** Returns a React element. */
-			return Object.setPrototypeOf(
```

Comment:
The proptotype of the forwarded ref object was previously set `Object(defaultType)`. This is not needed by React on the forwarded ref object, and AFAICT not needed by Stitches. 

`defaultType` is always the element/component passed in to the first `styled` call (when composing multiple styled calls), and is never the forward ref object returned by styled with additional Stitches properties, so I don't think this prototype makes a difference to Stitches when composing styled calls, but I didn't dig into the code behind `sheet.css(...)`. This passed tests and I tested it out in the apps that I have that use Stitches, and nothing popped up. So I think it's safe to delete.